native/cocos/platform/linux/modules/SystemWindow.cpp

```diff
@@ -75,10 +75,24 @@ bool SystemWindow::createWindow(const char *title,
     return true;
 }
 
+void SystemWindow::closeWindow() {
+    #ifndef CC_SERVER_MODE
```

Comment:
Is it possible to use SDL's interface to close a window on the windows platform?